content/rs/administering/import-export/database-backup.md

```diff
@@ -34,13 +37,232 @@ Redis Software copies a backup file for each shard to the specified backup locat
+
+If the previous backup fails, the trigger process retries the backup until it succeeds.
+
+In addition, throttling and resource limits also affect backup times.
+
+For help with specific backup issues, [contact support](https://redis.com/company/support/).
+
+
+## Supported storage locations {#supported-storage-locations}
+
+Database backups can be saved to a local mount point, transferred to [a URI](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier) using FTP/SFTP, or stored on cloud provider storage. 
+
+When saved to a local mount point or a cloud provider, backup locations need to be available to [the group and user]({{< relref "/rs/installing-upgrading/customize-user-and-group.md" >}}) running Redis Enterprise Software, `redislabs:redislabs` by default.  
+
+Redis Enterprise Software needs the ability to view permissions and update objects in the storage location. Implementation details vary according to the provider and your configuration. To learn more, consult the provider's documentation.
+
+The following sections provide general guidelines.  Because provider features change frequently, use your provider's documentation for the latest info.
+
+### FTP server
+
+Before enabling backups to an FTP server, verify that:
+
+- Your Redis Enterprise cluster can connect and authenticate to the FTP server.
+- The user specified in the FTP server location has read and write privileges.
+
+To store your backups on an FTP server, set its **Backup Path** using the following syntax:
+
+`ftp://[username]:[password]@[host]:[port]/[path]/`
+
+Where:
+
+- *protocol*: the server's protocol, can be either `ftp` or `ftps`.
+- *username*: your username, if needed.
+- *password*: your password, if needed.
+- *hostname*: the hostname or IP address of the server.
+- *port*: the port number of the server, if needed.
+- *path*: the backup path, if needed.
+
+Example: `ftp://username:password@10.1.1.1/home/backups/`
+
+The user account needs permission to write files to the server.
+
+### SFTP server
+
+Before enabling backups to an SFTP server, make sure that:
+
+- Your Redis Enterprise cluster can connect and authenticate to the SFTP server.
+- The user specified in the SFTP server location has read and write privileges.
+- The SSH private keys are specified correctly.  You can use the key generated by the cluster or specify a custom key.
+
+    When using the cluster auto generated key, copy the **Cluster SSH Public Key** to the appropriate location on the SFTP server.  This is available from the **General** tab of the **Settings** menu in the admin console.
+
+    Use the server documentation to determine the appropriate location for the SSH Public Key.
+
+To backup to an SFTP server, enter the SFTP server location in the format:
+
+```sh
+sftp://user:password@host:<:custom_port>/path/
```

Comment:
```suggestion
sftp://user:password@host:<custom_port>/path/
```

Reply:
Fixed-ish.  (There should only be one colon).